provider/pkg/provider/yaml/v2/yaml.go

```diff
@@ -77,6 +79,7 @@ func Parse(ctx context.Context, clientSet *clients.DynamicClientSet, opts ParseO
 				if err != nil {
 					return nil, errors.Wrapf(err, "expanding glob")
 				}
+				sort.Strings(files)
```

Comment:
Does expand require these to be sorted?

Reply:
No, I chose to sort the files to have the globbed manifests be visited in a stable order.  The K community sometimes uses `001-crds.yaml`, `002-deployment.yaml` etc. 

To be clear, this doesn't bear on the dependencies and is just a 'best effort'.